docs/hsi.xml

```diff
@@ -259,6 +267,38 @@
       </para>
     </refsect2>
 
+    <refsect3 id="org.fwupd.hsi.Uefi.SecureBoot">
+      <title>UEFI SecureBoot</title>
+      <para>
+        UEFI Secure boot is a verification mechanism for ensuring that code
+        launched by firmware is trusted.
+      </para>
+      <para>
+        Secure use of UEFI Secure Boot requires that each binary loaded at
+        boot is validated against trusted certifictes.
+      </para>
+      <itemizedlist>
+        <listitem>
+          <para>
+            For HSI-1 SecureBoot must be available for use on the platform.
```

Comment:
Although it's probably implied, you should say for HSI-1 on UEFI systems

Reply:
Do we care about non-UEFI systems? CSM boot should be an HSI failure, surely?

Reply:
Well you can ask the linuxboot, coreboot and oreboot guys that and you'll get a different answer than me :)

Reply:
> linuxboot, coreboot and oreboot

These all imply SB=off, right? None of them provide a chain of trust with a PK, right?

Reply:
Secure boot in the context described is a UEFI construct.  All these other things would have a _different_ trust chain approach.